Apps/Common/SIMSolver.h

```diff
@@ -195,12 +196,36 @@ template<class T1> class SIMSolver : public SIMadmin
     return restartStep;
   }
 
+  //! \brief Handles application data output.
```

Comment:
obviously this will be visible in downstreams, but who is now responsible for invoking this ? the protected nature of exporter implies the solver, but i do not see it doing so hence the question.